base/logging.go

```diff
@@ -75,6 +190,15 @@ func ParseLogFlag(flag string) {
 	}
 }
 
+func (config *LogAppenderConfig) ValidateLogAppender() error {
+	//Fail validation if an appender contains a "rotation" sub document
+	// and no "logFilePath" appender property is defined
+	if config.Rotation != nil && config.LogFilePath == nil {
+		return fmt.Errorf("The appender must define a \"logFilePath\" when \"rotation\" is defined")
```

Comment:
Changed to "The default logger must define \"logFilePath\" when \"rotation\" is defined"